src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/_help.py

```diff
@@ -188,57 +107,26 @@
 helps['managed-network managed-network-peering-policies list'] = """
     type: command
     short-summary: The ListByManagedNetwork PeeringPolicies operation retrieves all the Managed Network Peering Policies in a specified Managed Network, in a paginated format.
-    examples:
-      - name: Get Managed Network Group
-        text: |-
-               az managed-network managed-network-peering-policies list --managed-network-name \\
-               "myManagedNetwork" --resource-group "myResourceGroup"
 """
 
 helps['managed-network managed-network-peering-policies show'] = """
     type: command
     short-summary: The Get ManagedNetworkPeeringPolicies operation gets a Managed Network Peering Policy resource, specified by the  resource group, Managed Network name, and peering policy name
-    examples:
-      - name: Get Managed Network Peering Policy
-        text: |-
-               az managed-network managed-network-peering-policies show --managed-network-name \\
-               "myManagedNetwork" --managed-network-peering-policy-name "myHubAndSpoke" --resource-group \\
-               "myResourceGroup"
 """
 
 helps['managed-network managed-network-peering-policies create'] = """
     type: command
     short-summary: The Put ManagedNetworkPeeringPolicies operation creates/updates a new Managed Network Peering Policy
-    examples:

```

Comment:
seems all examples disappeared, I assume they will require additional fix?

Reply:
yes, Changlong is working on it